pkg/arvo/app/dojo.hoon

```diff
@@ -1055,6 +1057,11 @@
     ::  ~&  [%his-clock ler.cal]
     ::  ~&  [%our-clock ven.say]
     =^  dat  say  (~(transceive sole say) cal)
+    ::  speedrun mode
+    ::  =;  res
+    ::    ?.  ?=(%del -.dat)
+    ::      res
+    ::    (he-tab:res +(p.dat))
```

Comment:
Is this just auto-tab on input? Any reason to include it?

Reply:
yeah.  it's exhilarating, but impractical; removed.

Reply:
Something to remember, maybe it should be a configurable mode. For fun, if nothing else.